definitions/json/AuthorisationCaseField.json

```diff
@@ -731,18 +647,6 @@
     "UserRole": "et-acas-api",
     "CRUD": "R"
   },
-  {
-    "CaseTypeId": "ET_EnglandWales",
-    "CaseFieldID": "ethosCaseReference",
```

Comment:
should be kept as it's a way of identifying the case if the rep needs to contact the tribunal